properties/src/main/java/gov/cms/ab2d/properties/utils/Validation.java

```diff
@@ -0,0 +1,35 @@
+package gov.cms.ab2d.properties.utils;
+
+import org.apache.commons.lang3.StringUtils;
+
+public final class Validation {
```

Comment:
I am not seeing this class used anywhere (except for the test class).  If it is not used, please delete.  Otherwise, please illuminate where it is in use.

Reply:
See above comment by Keane.

Reply:
It's still useful to any users of the service.

Reply:
If it was useful, then it would be in use.  Doesn't appear to pass YAGNI to me.